css/style.css

```diff
@@ -162,13 +164,12 @@ td.moh-column-five {
 .select-style-second {
     padding: 0;
     margin: 10px 15px;
-    width: 40%;
     border: 1px solid #ccc;
-    width: 220px;
+  	width: 320px; 
     border-radius: 3px;
     overflow: hidden;
     background-color: #fff;
-    background: #fff url("http://www.scottgood.com/jsg/blog.nsf/images/arrowdown.gif") no-repeat 90% 50%;
+  /*  background: #fff url("http://www.scottgood.com/jsg/blog.nsf/images/arrowdown.gif") no-repeat 90% 50%; */
```

Comment:
Just delete - no need to comment out, we can get it from the github history if we need to  - though we never should since this is referencing an image on some other site (!)

Reply:
Yes I thought the reference was strange!